src/instructlab/utils.py

```diff
@@ -604,10 +604,7 @@ def is_oci_repo(repo_url: str) -> bool:
     Checks if a provided repository follows the OCI registry URL syntax
     """
     oci_url_regex = r"^docker://([a-zA-Z0-9\-_.]+(:[0-9]+)?)(/[a-zA-Z0-9\-_.]+)+(@[a-zA-Z0-9]+|:[a-zA-Z0-9\-_.]+)?$"
-    if not re.match(oci_url_regex, repo_url):
-        logger.warning(f"Invalid OCI repository URL: {repo_url}")
```

Comment:
we are losing some information surfacing with this change

Reply:
add a first check